app/templates/main/date-filters.html

```diff
@@ -96,5 +96,6 @@
     <button type="submit"
             class="govuk-button govuk-button__browse-all-filters-form-apply-button"
             data-module="govuk-button">Apply filters</button>
-    <a href="{% if view_name == 'browse' %} {{ browse_url }} {% elif view_name == 'transferring_body' %} {{ transferring_body_url ~ transferring_body_id }} {% elif view_name == 'series' %} {{ series_url ~ series_id }} {% elif view_name == 'consignment' %} {{ consigmnent_url ~ consignment_id }} {% endif %}">clear filters</a>
+    <a class="govuk-link govuk-link--browse-all-filter"
```

Comment:
I feel like the apply filters and clear filters html shouldnt be part of this date_filters template.

Also, I think we can simplify this href by doing something like 

```
<a href="{% if sorting_order %}{{ url_prefix ~ view_id ~ sorting_order }}{% else %}{{ url_prefix ~ view_id }}{% endif %}">Clear filters</a>
```

Reply:
removed apply filters and clear filters div from date-filters.html, added in individual browse view - all, transferring body and series view, will update for consignment view in AYR - 664 ticket